### PR TITLE
new: add murex shell support

### LINKED ISSUE #82: [Bounty] Add murex shell support
Add murex implementation to [starbase_shell](https://github.com/moonrepo/starbase/tree/master/crates/shell).

Docs: https://murex.rocks/

### PR DESCRIPTION
**Pull request by abhishek818**
fixes #82 
/claim #82

### CODE REVIEW

**Comment by abhishek818** [comment]
Up for review @milesj

File: crates/shell/src/shells/murex.rs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    fn get_env_path(&self, home_dir: &Path) -> PathBuf {
+        home_dir.join(".murex_preload")
+    }
+
+    fn get_profile_paths(&self, home_dir: &Path) -> Vec<PathBuf> {
+        vec![
+            home_dir.join(".murex_preload"),
+            home_dir.join(".murex_modules/"),
```
Comment (milesj) [inline crates/shell/src/shells/murex.rs:35]: Let's remove this since this list needs to be files only.

**Comment by milesj** [comment]
@abhishek818 Can you rebase master. Some of this method names changed, and new ones were added.

File: crates/shell/src/shells/murex.rs (hunk 1 above)
Comment (abhishek818) [reply crates/shell/src/shells/murex.rs]: fixed..

**Comment by milesj** [comment]
Thanks for the work!

**Comment by abhishek818** [comment]
> @milesj: The claim has been successfully added to reward-all. You can [visit your dashboard](https://console.algora.io) to complete the payment.

@milesj hey, seems bounty payment is not done. Ideally it should tag contributor's username and send a email notification also.
Can you check this [this](https://docs.algora.io/bounties/workflow#pay-out-a-claim

**Comment by milesj** [comment]
Ok manually did it in the console, should be good now.

**Comment by abhishek818** [comment]
@milesj yup, that worked thanks.

**Pull request merged**